repositories/repositories.bzl

```diff
@@ -219,6 +219,14 @@ py_library(
             urls = ["https://storage.googleapis.com/container-structure-test/v1.8.0/container-structure-test-darwin-amd64"],
         )
 
+    if "container_diff" not in excludes:
```

Comment:
you probably also want to add base_images_docker here instead of in WORKSPACE

Reply:
Done